internal/components/receivers_test.go

```diff
@@ -93,7 +93,7 @@ func TestDefaultReceivers(t *testing.T) {
 		},
 	}
 
-	assert.Equal(t, len(tests)+28 /* not tested */, len(rcvrFactories))
+	assert.Equal(t, len(tests)+29 /* not tested */, len(rcvrFactories))
```

Comment:
This should be avoided. Please, try to add the component to the set of components under test.